styles.css

```diff
@@ -0,0 +1,974 @@
+}
+
+li.social-icon {
+  display: flex;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  width: 90%;
+  height: 600px;
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 8%;
+}
+
+.title {
+  position: static;
+  width: 280px;
+  height: 25%;
+  left: 0;
+  top: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding-top: 10px;
+  padding-bottom: 20px;
+}
+
+.role {
+  display: grid;
+  grid-template-columns: 1fr 0.5fr 1.5fr 0.5fr 1.5fr 0fr;
+}
+
+.role-name {
+  position: static;
+  width: 55px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 2px;
+}
+
+.role-dot {
+  width: 10px;
+  height: 10px;
+  left: 67px;
+  top: 12px;
+  background: #c1c7d0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 2.5px 12px;
+  border-radius: 50%;
+}
+
+.role-job {
+  position: static;
+  width: 88px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.role-year {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.project-description {
+  margin-left: 5%;
+}
+
+.project-description > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  height: 90px;
+}
+
+.project-details {
+  width: 70%;
+  height: 76px;
+  left: 0;
+  top: 92px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.skills {
+  display: flex;
+  float: right;
+  align-items: center;
+  position: static;
+  padding: 0.5px;
+  width: 100%;
+  height: 24px;
+  left: 0.5px;
+  top: 180px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-top: 10%;
+}
+
+li.skill {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 4px 12px;
+  position: static;
+  width: 53px;
+  height: 24px;
+  left: 0.5px;
+  top: 0.5px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 8px;
+}
```

Comment:
- [ ] You really put in the work, however, kindly make this final change to get approved. You need to adjust the technology area cover to have the same dimensions as that of the Figma template. 
- The space gap for the `HTML and CSS` looks a little more than that on the Figma, and does not make the skill section all aligned 
- This might look normal t you, but paying attention to such details matters a lot when working with clients of product managers in the future 👍 

## Figma:
![image](https://user-images.githubusercontent.com/50990925/228756780-4f1f96e4-790d-436e-8004-e022f7bd8325.png)

## your view:
![image](https://user-images.githubusercontent.com/50990925/228757114-2ac8360d-8949-49db-afb8-dc9db090996b.png)